apps/nrm/translations/src/en/fields.json

```diff
@@ -547,17 +547,19 @@
     "paragraph-1": "If a potential victim wants you to refer their case, they can get:",
     "bullet-1": "access to support (for example, housing and legal advice)",
     "bullet-2": "a formal decision – this can help if they’re being prosecuted for a crime they were forced to commit",
-    "div-1-paragraph-1": "We will need to share their personal details with organisations such as the police and the Home Office if they agree to their case being referred.",
-    "div-1-paragraph-2": "Please explain to the potential victim the ways in which the Home Office may process their personal information, including the purposes for which it will be used, the legal basis and who it will be shared with. This is contained in the <a class=\"govuk-link\" href=\"https://www.gov.uk/government/publications/human-trafficking-victims-referral-and-assessment-forms/privacy-information-notice-national-referral-mechanism\">Privacy Information Notice</a> and <a class=\"govuk-link\" href=\"https://www.gov.uk/government/organisations/home-office/about/personal-information-charter\">Personal Information Charter.</a>",
+    "nrm-checks-p": "You should:",
```

Comment:
maybe this property name could be paragraph-2  since it's the next `p` tag on the page